tran_kien_giang/extend.php

```diff
@@ -0,0 +1,46 @@
+<?php 
+	$so_tien_can_quy_doi=$_GET['so_tien_can_quy_doi'];
```

Comment:
```suggestion
	$soTienCanQuyDoi = $_GET['so_tien_can_quy_doi'];
```
tên biến kiểu Camelcase